src/utils/date.ts

```diff
@@ -0,0 +1,25 @@
+export const formatYYYYMMDD = (date?: Date) =>
```

Comment:
We have all of this date utils in date-fns library

Reply:
updated